doc/introduction/pulse.rst

```diff
@@ -0,0 +1,289 @@
+                 0.        -0.23993255j,  0.        -0.95982337j],
+               [ 0.        -0.95982337j,  0.        +0.23993255j,
+                -0.14115842+0.j        ,  0.03528605+0.j        ],
+               [ 0.        -0.23993255j,  0.        -0.95982337j,
+                -0.03528605+0.j        , -0.14115842+0.j        ]],      dtype=complex64)
+
+
+The parameters can be updated by calling the :class:`~.ParametrizedEvolution` again with different inputs.
+
+Additional options with regards to how the matrix is calculated can be passed to the :class:`.ParametrizedEvolution`
+along with the parameters, as keyword arguments:
+
+.. code-block:: python
+
+    qml.evolve(H)(params=[1.2], t=[0, 4], atol=1e-6, mxstep=1)
+
+These options are:
+
+- `atol (float, optional)`: Absolute error tolerance
+- `rtol (float, optional)`: Relative error tolerance
+- `mxstep (int, optional)`: maximum number of steps to take for each time point
+- `hmax (float, optional)`: maximum step size allowed
+
+If not specified, they will default to predetermined values. See :class:`.ParametrizedEvolution` for details.
+
+Using qml.evolve in a QNode
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The :class:`~.ParametrizedEvolution` can be implemented in the QNode in the same way as any other
+PennyLane :class:`~.Operator`.
+
+To look at an example of this, let's start with two instances of :class:`~.ParametrizedHamiltonian`:
+
+.. code-block:: python
+
+    ops = [qml.PauliX(0), qml.PauliY(1), qml.PauliZ(2)]
+    coeffs = [lambda p, t: p for _ in range(3)]
+    H1 = qml.ops.dot(coeffs, ops)  # time-independent parametrized hamiltonian
+
+.. code-block:: python
+
+    ops = [qml.PauliZ(0), qml.PauliY(1), qml.PauliX(2)]
+    coeffs = [lambda p, t: p * jnp.sin(t) for _ in range(3)]
+    H2 = qml.ops.dot(coeffs, ops) # time-dependent parametrized hamiltonian
+
+Now we can execute the evolution of these Hamiltonians applied simultaneously:
+
+.. code-block:: python
+
+    dev = qml.device("default.qubit", wires=3)
+    @qml.qnode(dev, interface="jax")
+    def circuit(params):
+        qml.evolve(H1 + H2)(params, t=[0, 10])
+        return qml.expval(qml.PauliZ(0) @ qml.PauliZ(1) @ qml.PauliZ(2))
+
+    params = jnp.array([1., 2., 3., 4., 5., 6.])
+    circuit(params)
+    >>> Array(-0.78236955, dtype=float32)
+
+
+.. warning::
+    In this example, it is important that ``H1`` and ``H2`` are included in the same ``qml.evolve`` operation.
+    For non-commuting operations, applying ``qml.evolve(H1)(params, t=[0, 10])`` followed by
+    ``qml.evolve(H2)(params, t=[0, 10])`` will NOT apply the two pulses simultaneously, despite the overlapping
+    time window. Instead, it will execute ``H1`` in the ``[0, 10]`` time window, and then subsequently execute
+    ``H2`` using the same time window to calculate the evolution, but without taking into account how the time
+    evolution of ``H1`` affects the evolution of ``H2`` and vice versa.
```

Comment:
Very nice, and important! Maybe the first sentence could state that it is important not for technical reasons but that it is important in order to really implement the simultaneous evolution.
I.e. the sequence of `qml.evolve(H1)` and `qml.evolve(H2)` also is valid, it's just physically not what we wanted to implement in this example...

Reply:
I rephrased the beginning a bit, let me know what you think!

Reply:
It's very clear now, thanks a lot! :)